### PR TITLE
[Makefile/docker-compose.yaml] set SOURCE_BRANCH

### PR DESCRIPTION
**Pull request by jpopelka**
This was (accidentally?) removed in daf86eed8

### CODE REVIEW

**Comment by IceWreck** [comment]
Can you please add something like this to docker-compose as well
```
build:
      context: .
      dockerfile: files/docker/Dockerfile.worker
      args:
        SOURCE_BRANCH: master

```
In service and worker

**Comment by csomh** [comment]
I'm also wondering weather it would make sense to have a default of `master` in the Dockerfiles.... it's weird when someone builds directly with docker and then finds out that there was a required build argument.

File: Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ AP ?= ansible-playbook -vv -c local -i localhost, -e ansible_python_interpreter=
 PATH_TO_SECRETS ?= $(CURDIR)/secrets/
 COV_REPORT ?= term-missing
 COLOR ?= yes
-SOURCE_BRANCH ?=
+SOURCE_BRANCH ?= $(shell git branch --show-current)
```
Comment (TomasTomecek) [inline Makefile:12]: actually, now that I think about this, I second Hunor, shouldn't we hardcode "master" here?

will the build even work if it's something which is not stable nor master?

Comment (jpopelka) [reply Makefile:12]: https://github.com/packit-service/packit-service/blob/master/files/tasks/process-source-branch.yaml#L9

Oh come on guys, this is a hotfix PR, not a new feature PR :-)
Let me merge this and leave this to @sakalosj :-)

**Comment by jpopelka** [comment]
> I'm also wondering weather it would make sense to have a default of `master` in the Dockerfiles

I don't know why @sakalosj  explicitly [fails on SOURCE_BRANCH being empty](https://github.com/packit-service/packit-service/blob/master/files/tasks/process-source-branch.yaml#L6), so I'd rather let him answer this.
As I said, this is just a hotfix to unblock @IceWreck

**Comment by sakalosj** [comment]
> > I'm also wondering weather it would make sense to have a default of `master` in the Dockerfiles
> 
> I don't know why @sakalosj explicitly [fails on SOURCE_BRANCH being empty](https://github.com/packit-service/packit-service/blob/master/files/tasks/process-source-branch.yaml#L6), so I'd rather let him answer this.
> As I said, this is just a hotfix to unblock @IceWreck

@jpopelka @IceWreck @csomh @TomasTomecek 
I decided to not default to any value is to avoid situations when because of some error or misconfiguration we can end up deploying master also to prod environment. From my perspective, this behavior is oriented toward automation - we should never achieve state when SOURCE_BRANCH is not defined. 

My suggestion is to implement a check if SOURCE_BRANCH is defined into Dockerfile so it fails fast if not defined and implement this into Dockerfile too, but I am open to any solution.

**Pull request merged**